docs/docs/adapters/overview.md

```diff
@@ -0,0 +1,74 @@
+# Adapters
+
+The objective of Mopro is to support a wide range of circuits and platforms. To achieve this, Mopro provides adapters
+that allow you to build proofs for your circuits and do other cryptographic operations on different platforms. The
+adapters are designed to be modular and easy to use.
+
+## Overview
+
+Mopro's modular architecture allows you to combine different adapters in your project, such as mixing Halo2 and Circom
+circuits in the same project. The adapters are designed to be easy to use and to provide a consistent interface across
+different platforms.
+
+## Enabling Adapters
+
+To activate a specific adapter, you must enable it in your library's `Cargo.toml` file using the `mopro-ffi` feature
+mechanism. Each adapter has its own feature name, along with a list of dependencies that need to be included in your
+project.
+
+For example, to enable the Circom adapter, ensure that the `mopro-ffi/circom` feature is enabled in your `Cargo.toml`
+file:
+
+```toml
+[features]
+default = ["mopro-ffi/circom"]
+```
+
+To mix different adapters, you can enable multiple features:
+
+```toml
+[features]
+default = ["mopro-ffi/circom", "mopro-ffi/halo"]
```

Comment:
haven't been solved here?

Reply:
Did a regex search, that is the last one